core/consensus/babe/types/epoch_descriptor.hpp

```diff
@@ -31,15 +29,17 @@ namespace kagome::consensus {
             typename = std::enable_if_t<Stream::is_encoder_stream>>
   Stream &operator<<(Stream &s, const EpochDescriptor &led) {
     return s << led.epoch_number << led.start_slot
-             << 0LL;
+             << static_cast<int64_t>(0)  // TODO(xDimon): Remove it after wipe
```

Comment:
When is this going to happen?

Reply:
Decided to do it after PR will be merged